presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConnectorModule.java

```diff
@@ -36,9 +43,12 @@
     @Override
     public void configure(Binder binder)
     {
-        binder.bind(KafkaMetadata.class).in(Scopes.SINGLETON);
-        binder.bind(KafkaSplitManager.class).in(Scopes.SINGLETON);
-        binder.bind(KafkaRecordSetProvider.class).in(Scopes.SINGLETON);
+        binder.bind(ConnectorMetadata.class).to(KafkaMetadata.class).in(Scopes.SINGLETON);
+        binder.bind(Key.get(ConnectorSplitManager.class, ForClassLoaderSafe.class)).to(KafkaSplitManager.class).in(Scopes.SINGLETON);
```

Comment:
The canonical way is
```java
binder.bind(ConnectorSplitManager.class).annotatedWith(ForClassLoaderSafe.class).to(KafkaSplitManager.class).in(Scopes.SINGLETON);

Reply:
> The canonical way is

I didn't know. Let me help memoize this :) https://github.com/prestosql/presto/pull/2483

Reply:
"canonical" refers to the usage of `Key.get` vs `annotatedWith`, but we've historically formatted them across multiple lines when they get that long (I think it's more readable that way, too).